.github/workflows/go-scheduled.yml

```diff
@@ -77,13 +77,12 @@ jobs:
     strategy:
       matrix:
         test-setups:
-        - {name: "Multinode",                             ext: true,  tsdb: true,  tsdb2: true,  multi: true,  pg: 13, nightly: false}
-        - {name: "W/O Promscale Extension",               ext: false, tsdb: true,  tsdb2: true,  multi: false, pg: 13, nightly: false}
-        - {name: "Plain Postgres",                        ext: false, tsdb: false, tsdb2: false, multi: false, pg: 13, nightly: false}
-        - {name: "Timescaledb 1.x (pg 12)",               ext: true,  tsdb: true,  tsdb2: false, multi: false, pg: 12, nightly: false}
-        - {name: "Plain Postgres (12)",                   ext: false, tsdb: false, tsdb2: false, multi: false, pg: 12, nightly: false}
-        - {name: "TimescaleDB Nightly (PG-13)",            ext: false, tsdb: true,  tsdb2: true,  multi: false, pg: 13, nightly: true}
-        - {name: "TimescaleDB-Multinode Nightly (PG-13)",  ext: false, tsdb: true,  tsdb2: true,  multi: true,  pg: 13, nightly: true}
+        - {name: "Singlenode (12)",                     tsdb: true,  multi: false, pg: 12, nightly: false}
+        - {name: "Singlenode (14)",                     tsdb: true,  multi: false, pg: 14, nightly: false}
+        - {name: "Without TimescaleDB",                 tsdb: false, multi: false, pg: 14, nightly: false}
```

Comment:
If TSDB is mandatory I wonder why we keep e2e tests without it?

Reply:
Promscale extension is mandatory, Timescale extension is not.

Reply:
what ante said